Cmake error Patching leveldb

After a major system upgrade to the recent Tumbleweed version I had to recompile the CVMFS client.
Source:
https://github.com/cvmfs/cvmfs/archive/refs/heads/cvmfs-2.10.zip

`cmake ../` fails with the following error message:
```
--> Patching leveldb
patching file build_detect_platform
patching file build_detect_platform
Hunk 1 FAILED 185/185.
     fi
 
     # Test whether tcmalloc is available
-    $CXX $CXXFLAGS -x c++ - -o $CXXOUTPUT -ltcmalloc 2>/dev/null  <<EOF
-      int main() {}
-EOF
-    if [ "$?" = 0 ]; then
-        PLATFORM_LIBS="$PLATFORM_LIBS -ltcmalloc"
-    fi
+#    $CXX $CXXFLAGS -x c++ - -o $CXXOUTPUT -ltcmalloc 2>/dev/null  <<EOF
+#      int main() {}
+#EOF
+#    if [ "$?" = 0 ]; then
+#        PLATFORM_LIBS="$PLATFORM_LIBS -ltcmalloc"
+#    fi
 
     rm -f $CXXOUTPUT 2>/dev/null
 fi
CMake Error at CMakeLists.txt:208 (message):
  bootstrapping failed
```

  
To solve this issue I had to disable leveldb in the cmake configuration.

--- a/externals/leveldb/src/dont_search_tcmalloc.patch
+++ b/externals/leveldb/src/dont_search_tcmalloc.patch
@@ -1,7 +1,7 @@
---- build_detect_platform.orig	2013-08-15 13:32:42.000000000 +0200
-+++ build_detect_platform	2013-08-15 13:36:26.000000000 +0200
-@@ -185,12 +185,12 @@
-     fi
+--- build_detect_platform.orig	2022-11-21 11:50:01.933197574 +0100
++++ build_detect_platform	2022-11-21 11:50:52.179862263 +0100
+@@ -202,12 +202,12 @@
+ #    fi
  
      # Test whether tcmalloc is available
 -    $CXX $CXXFLAGS -x c++ - -o $CXXOUTPUT -ltcmalloc 2>/dev/null  <<EOF

